openedx/core/djangoapps/enrollments/views.py

```diff
@@ -699,9 +699,23 @@ def post(self, request):
 
         has_api_key_permissions = self.has_api_key_permissions(request)
 
-        # Check that the user specified is either the same user, or this is a server-to-server request.
+        # Check if the user or email was defined.
```

Comment:
Nit: Can we add that note about priorities of which input takes precedence here? Maybe something like:

```suggestion
        # A provided user has priority over a provided email.
        # Fallback on request user if neither is provided.
```